react-instastories.js.org

kenclaron opened the pull request:
<!--

Thanks for creating a pull request to request a new subdomain from JS.ORG

Before creating your pull request, please complete the following steps:

- Ensure that your pull request changes only the cnames_active.js file, adding a single new line for your subdomain request
- Tick the two checkboxes, agreeing to the sentences, below by placing an x inside the square brackets ([ ] becomes [x])
- Add a link (GitHub repository, Vercel deployment, etc.) and explanation below for your content so we can validate your request

-->

- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- [x] I have read and accepted the [Terms and Conditions](http://js.org/terms.html)
- The site content can be seen at https://kenclaron.github.io/react-instastories/

> The site content is about npm packages: [@react-instastories/base](https://www.npmjs.com/package/@react-instastories/base), [@react-instastories/external](https://www.npmjs.com/package/@react-instastories/external), [@react-instastories/presets](https://www.npmjs.com/package/@react-instastories/presets)

MattIPv4 reviewed (changes requested):
Hi there!

❌ Before we can accept your domain request, you need to configure your GitHub Pages site to use the requested js.org subdomain. Head to your repository settings, locate the Pages settings (under Code and Automation), and enter the js.org subdomain in the custom domain field. 📝

kenclaron commented:
> Hi there!
> 
> ❌ Before we can accept your domain request, you need to configure your GitHub Pages site to use the requested js.org subdomain. Head to your repository settings, locate the Pages settings (under Code and Automation), and enter the js.org subdomain in the custom domain field. 📝

Hi, fixed. I added `react-instastories.js.org` in Pages - Custom domain

indus commented:
Welcome https://react-instastories.js.org
(FYI: looks like some assets are not getting loaded because the basepath has changed)
